.circleci/config.yml

```diff
@@ -392,6 +402,9 @@ workflows:
             - test-exchange-ganache-3.0:
                   requires:
                       - build
+            - test-integrations-ganache-3.0:
+                  requires:
+                      - build
```

Comment:
Not sure why this is showing up as a diff, but I can't get rid of it. This file is identical to the file in `development`.